src/worldcereal/openeo/feature_extractor.py

```diff
@@ -38,7 +39,7 @@ class PrestoFeatureExtractor(PatchFeatureExtractor):
         "S2-L2A-B12": "B12",
         "S1-SIGMA0-VH": "VH",
         "S1-SIGMA0-VV": "VV",
-        "COP-DEM": "DEM",
+        "COP-DEM": "elevation",
```

Comment:
In fact, we assume `DEM-alt-20m` which is later remapped to `elevation`: https://github.com/WorldCereal/presto-worldcereal/blob/main/presto/inference.py#L66

However, this may not be consistent with the other gfmap naming schemes. So it's fine by me to keep it like this for now, but I'll need to make a change in the presto code for that.